scripts/core/utils.ts

```diff
@@ -213,3 +213,15 @@ export function isScrolledIntoViewVertically(element: HTMLElement, container: HT
 
     return topVisible && bottomVisible;
 }
+
+export function downloadFile(data: {}, fileName: string, extensionValue: string, htmlElementId: string) {
```

Comment:
Can you drop `htmlElementId` argument, and create an element in this function instead? Also avoid using jQuery where it's easy to do so.